docs/session-state/remote-session.md

```diff
@@ -8,38 +8,50 @@ In order to configure it, both the framework and core app must set an API key as
 
 - `ApiKeyHeader` - header name that will contain an API key to secure the endpoint added on .NET Framework
 - `ApiKey` - the shared API key that will be validated in the .NET Framework handler
-- `RegisterKey<T>(string)` - Registers a session key to a known type. This is required in order to serialize/deserialize the session state correctly. If a key is found that there is no registration for, an error will be thrown and session will not be available.
```

Comment:
Nit-pick, it looks like we removed the explanation of `RegisterKey` here but didn't add it anywhere else as serialization options. It's an important concept, so we should make sure to document it.

Reply:
I thought I had moved it to the serialization section. I've pushed a change with that.